e2e/cypress/integration/integrations/message_to_channel_via_slash_command_spec.js

```diff
@@ -21,8 +21,8 @@ describe('Integrations', () => {
         cy.requireWebhookServer();
 
         // # Create new team and get off-topic channel
-        cy.apiCreateTeam('test-team', 'Test Team').then((teamResponse) => {
-            team = teamResponse.body;
+        cy.apiCreateTeam('test-team', 'Test Team').then(({team: newTeam}) => {
+            team = newTeam;
```

Comment:
Shouldn't this be simply set as:
`cy.apiCreateTeam('test-team', 'Test Team').then((team) => {`

Reply:
See above response.